README.md

```diff
@@ -2,7 +2,11 @@
 
 # Community Support
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](http://discord.eddiehub.org)
+
+<p align="center">
+ <img src= "https://img.shields.io/discord/699608417039286293?style=flat-square)](http://discord.eddiehub.org" >
```

Comment:
Link is not working. This would be a suitable solution for this.

```suggestion
 <a href="http://discord.eddiehub.org"><img src= "https://img.shields.io/discord/699608417039286293?style=flat-square)"></a>
```

Reply:
I think that's only a suitable solution which I have used because I have tried many combinations but the link isn't working. Lastly it works on that code which I have commited